Commands/SpawnBoss.cs

```diff
@@ -0,0 +1,102 @@
+﻿using ProjectM;
+using Unity.Entities;
+using VampireCommandFramework;
+using System;
+using Unity.Mathematics;
+using Unity.Transforms;
+using CommunityCommands.Commands;
+
+namespace CommunityCommands.Commands
+{
+    internal static class SpawnBoss
+    {
+        [Command("spawnboss", "sb", description: "Spawn a boss next to the player", adminOnly: true)]
+        public static void SpawnEnemy(ChatCommandContext ctx, string bossName, int quantity = 1)

```

Comment:
I think a lot of this might be simplified if you made an enum `BossName` that had all the names plus `all`. Alternatively you could make something like `record struct BossName(string name)` and a `class BossNameConverter : CommandArgumentConverter<BossName>` that did the lookup with whatever partial/best matching.